src/gpt2/gpt2_runner.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, model, optimizer, data):
         self.model = GPT2LMHeadModel(AutoConfig.for_model(**model))
         self.tokenizer = GPT2TokenizerFast.from_pretrained(self.hparams.data.tokenizer_dir, pad_token="<|endoftext|>")
         self.model.resize_token_embeddings(len(self.tokenizer))
-        self.collate_fn = DataCollatorForCausalLanguageModeling(
+        self.collate_fn = DataCollatorForPretrainingGPT2(
```

Comment:
```suggestion
        self.collate_fn = DataCollatorForPretrainingGPT2(tokenizer=self.tokenizer)
```

이거 그냥 올려 붙여도 될 것 같은데, formatter가 해준 것일까요?ㅋ

Reply:
일단 formatter의 준엄한 심판을 넘기긴했는데 아마 `self.tokenizer,` 이거로 되어있어서 형석님이 의도하신대로 된 것이 아닌듯합니다.